packages/rancher-istio-1.9/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-packageVersion: 1
+version: 20.0.0
```

Comment:
@deniseschannon should istio-1.9 be 20.10.0?

It cannot be the same as istio 1.8 or istio 1.10 (tbd in an upcoming PR) and we need 10 since a Rancher 2.7 release would cause the minor version to be bumped.